presto-main/src/main/java/io/prestosql/operator/OrderByOperator.java

```diff
@@ -56,6 +57,7 @@
         private final PagesIndex.Factory pagesIndexFactory;
         private final boolean spillEnabled;
         private final Optional<SpillerFactory> spillerFactory;
+        private final OrderingCompiler orderingCompiler;
```

Comment:
move this field after `private final List<SortOrder> sortOrder;`

(and move `private boolean closed;` to be last, separated by a blank line)

Reply:
I think that `orderingCompiler` is  tightly coupled with `spillerFactory`, so they can as well be paired as parameters (similar in window operator)

Moved `closed`